openfisca_us/variables/gov/states/mo/tax/income/adjusted_gross_income/mo_federal_itemized_deductions.py

```diff
@@ -0,0 +1,20 @@
+from openfisca_us.model_api import *
+
+
+class mo_federal_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Sum of Federal itemized deductions applicable to MO taxable income calculation"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://dor.mo.gov/forms/MO-A_2021.pdf"
```

Comment:
I'm not seeing this list here. Is it in the law?

Reply:
This is from page 11 of this document here https://dor.mo.gov/forms/4711_2021.pdf - I'll add it to the newest version of this file right now.

Reply:
here's the leg ref: